js-exercises/last-index-of/lastIndexOf.js

```diff
@@ -0,0 +1,14 @@
+function lastIndexOf(...args) {
+  if (args.length === 2 && Array.isArray(args[1])) {
+    for (let i = args[1].length - 1; i >= 0; i--) {
```

Comment:
Very minor, but it would definitely help with nomenclature & clarity if you stored args[1] in a variable called 'list' and element to be search in a variable called 'element'.